src/utils/api.js

```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
+
+export const getImage = async (setImageData) => {
+  try {
+    const {
+      data: { urls, location, discrptions }
```

Comment:
is it a typo?

Reply:
No ,here i did destructure accordingly which is  name given in data (api )